dvc/progress.py

```diff
@@ -62,18 +68,27 @@ def __init__(
         desc  : persists after `close()`
         level  : effective logging level for determining `disable`;
             used only if `disable` is unspecified
+        disable  : Overridden by env var TQDM_DISABLE; otherwise
+            if (default: None), will be determined by logging level
+            and TTY status.
         kwargs  : anything accepted by `tqdm.tqdm()`
         """
         kwargs = kwargs.copy()
         kwargs.setdefault("unit_scale", True)
+        kwargs.setdefault("file", sys.stderr)
         if bytes:
             bytes_defaults = dict(
                 unit="B", unit_scale=True, unit_divisor=1024, miniters=1
             )
             kwargs = merge(bytes_defaults, kwargs)
         self.desc_persist = desc
-        if disable is None:
-            disable = logger.getEffectiveLevel() > level
+        if TQDM_DISABLE is not None:  # env var override
+            disable = TQDM_DISABLE
```

Comment:
How do I enable it? I don't want disabling it. And this should not override `disable` flag, like in `disable = total == 0`, this should only override isatty, there should not appear skipped empty pbars from nowhere.

Also, negative flags complicate things.

Reply:
`TQDM_DISABLE=False` (see below comments)